pkg/dashboard/gitops/helper_test.go

```diff
@@ -94,6 +94,7 @@ func initHistory() *git.Repository {
 		map[string]string{
 			"file": `1`,
 		},
+		nil,
```

Comment:
These will be not needed if you use the `files` field for the kustomization per app file.